### PR TITLE
docs(portuguese): updating and adding missing translations

### PR DESCRIPTION
**Pull request by fernandohmg**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- Updated outdated pages
- Translations improvements and minor fixes

---

### CODE REVIEW

File: www/src/pages/pt/usage/env-variables.mdx [hunk 1]
```diff
@@ -62,27 +55,82 @@ Precisamos de um objeto JavaScript com o qual possamos analisar nossos esquemas
...
 ## Usando Variáveis Ambiente
 
-Quando você quiser usar suas variáveis de ambiente, você pode importá-las de `env/client.mjs` ou `env/server.mjs` dependendo de onde você deseja usá-las:
+Quando você quiser usar suas variáveis de ambiente, você pode importá-las de `env.mjs` e usá-las como faria normalmente. Se você importar isso no cliente e tentar acessar uma variável de ambiente do lado do servidor, receberá um erro em tempo de execução.
 
 ```ts:pages/api/hello.ts
 import { env } from "../../env.mjs";
 
-// `env` is fully typesafe and provides autocompletion
+// `env` é totalmente tipado e fornece autocompletar
```
Comment (Sn0wye) [inline www/src/pages/pt/usage/env-variables.mdx:122]:
```suggestion
// `env` é totalmente tipado e fornece autocomplete
```

**Review by Sn0wye — CHANGES_REQUESTED** [review]
Just this bit, I'd say mantaining it in English makes more sense.
Besides that, LGTM 🎉

File: www/src/pages/pt/usage/env-variables.mdx (hunk 1 above)
Comment (fernandohmg) [reply www/src/pages/pt/usage/env-variables.mdx]: That makes sense! Just added it

**Review by Sn0wye — APPROVED** [review]
Seems nice!

**Review by juliusmarminge — APPROVED** [review]

**Pull request merged**